standard/standard_2.php

```diff
@@ -971,7 +971,7 @@ function rand ($min = 0, $max = null) {}
  * an alias of {@see mt_srand()}.
  * </p>
  * @link https://php.net/manual/en/function.srand.php
- * @param int $seed [optional] <p>
+ * @param int|null $seed [optional] <p>
```

Comment:
This change doesn't correspond to the [docs](https://www.php.net/manual/en/function.srand.php) and the actual param type is `zend_long` which is `typedef int64_t zend_long;` (according to [sources](https://github.com/php/php-src/blob/623911f993f39ebbe75abe2771fc89faf6b15b9b/ext/standard/rand.c#L33)). Although the behavior of `srand(null)` seems legit (because of how C works), I don't see how this could improve code analysis and benefit the user who reads stub's doc.